tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ts

```diff
@@ -36,6 +36,11 @@ export enum Fob {
   END,
 }
 
+const TIME_SELECTION_TO_FOB: Record<keyof TimeSelection, Fob> = {
+  start: Fob.START,
+  end: Fob.END,
+};
```

Comment:
This is somewhat undesirable and should perhaps be a reason to abandon the `Fob` enum.

Reply:
Abandon it in favor of what? What would this.currentDraggingFob be?

Reply:
I would replace it with a type that naturally maps to `keyof TimeSelection` (maybe just use `keyof TimeSelection`). I looked at what that would take though and it is beyond the scope of this PR.